test/integration/targets/cronvar/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+- name: Create EMAIL cron var
+  cronvar:
+    name: EMAIL
+    value: doug@ansibmod.con.com
+  register: create_cronvar1
+
+- name: Create EMAIL cron var again
+  cronvar:
+    name: EMAIL
+    value: doug@ansibmod.con.com
+  register: create_cronvar2
+
+- name: Check cron var value
+  command: grep -c EMAIL=doug@ansibmod.con.com {{ cron_user_config_path }}/root
```

Comment:
How about something like `crontab -u root -l | grep 'EMAIL=doug@ansibmod.con.com'` instead? Same for line 24.

Reply:
Will do. Thank you for the suggestion. That makes things much simpler give that the file is in different locations on different distributions (which I was just fixing).